### DISCUSSION TITLE
Docs: Playwright Component Testing

### DISCUSSION THREAD

**Discussion by sand4rt** [vercel/next.js#69329] [Ideas]
### What is the documentation issue?

Can i add a section on Playwright component testing?

### Is there any context that might help us understand?

nope

### Does the docs page already exist? Please link to it.

https://nextjs.org/docs/app/building-your-application/testing/playwright

**Comment by samcx** [comment]
@sand4rt Feel free to push up a PR!

I moved this to discussions since it's technically not a :docs: issue and more so a request to add additional docs.
